src/sentry/tasks/unmerge.py

```diff
@@ -177,6 +177,9 @@ def migrate_events(caches, project, source_id, destination_id, fingerprints, eve
 
         destination_id = destination.id
 
+        # TODO: This blocks on having each event's full hash list as an array in ClickHouse
+        eventstream.unmerge(project.id, fingerprints, destination_id)
```

Comment:
This is the main thing I need reviewed ASAP because if it's the only path forward then I need to work on some other PRs to get this data flowing into CH.